### ISSUE TITLE
Accessibilité et qualité

### ISSUE THREAD

**Issue by MagNott** [MagNott/P9_Developpez_application_web_django#18]
**Avant la livraison :** 

- Vérifier l'accessibilité et mettre à jour le code (référentiel WCAG)
- Se conformer à Pep8 via flake ou black
- Mettre à jour le Readme
